composer.json

```diff
@@ -8,14 +8,23 @@
   "minimum-stability": "stable",
   "autoload": {
     "psr-4": {
-      "": "src/",
+      "Google\\Photos\\": "src/Google/Photos/",
+      "GPBMetadata\\Photos\\": "src/GPBMetadata/Google/Photos/"
```

Comment:
Not quite yet - this needs the full package name that's referenced in the path and the files within it, including `Google`.

So you'd want this:
```
      "GPBMetadata\\Google\\Photos\\": "src/GPBMetadata/Google/Photos/"
```

Reply:
would you object to me rebasing these commits so the merge isn't peppered with these corrections?

Reply:
Yes, feel free to squash these iterations together.  (I was going to squash them otherwise on final merge.)

Thanks again!